src/UpdatedPackagesCollector.js

```diff
@@ -52,52 +52,45 @@ export default class UpdatedPackagesCollector {
   collectUpdatedPackages() {
     this.logger.info("", "Checking for updated packages...");
 
-    const hasTags = GitUtilities.hasTags(this.execOpts);
+    const { execOpts, options } = this;
+    const { canary } = options;
+    let { since } = options;
 
-    if (hasTags) {
-      const tag = GitUtilities.getLastTag(this.execOpts);
-      this.logger.info("", "Comparing with tag " + tag);
-    } else {
-      this.logger.warn("", "No tags found!");
-      this.logger.info("", "Comparing with initial commit.");
-    }
+    if (GitUtilities.hasTags(execOpts)) {
```

Comment:
While this is a boyscout, it's also a fix for the lines I added to the test to make sure that `since` overrides the latest tag, if there are tags present.